### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by illusion2626**
### Pull Request Overview

This pull request adds the KiCad schematic of the project (Screenshot), a photo of the components used and a photo of the assembled project

### Testing Strategy

This pull request was tested by Boghici Ionut-Daniel

### TODO or Help Wanted

For the LCD display, I haven't 100% settled on how to fix it into place, so for now it's set loose on the breadboards

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Boghici Ionuț-Daniel <ionutdaniel264@gmail.com>

### CODE REVIEW

**Comment by alexandruradovici** [comment]
Write the description.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please move the kicad files to your project's repository.

**Comment by illusion2626** [comment]
> Please move the kicad files to your project's repository.

I moved them to my project's repository.

File: docs/project/2024/ionut.boghici/project.kicad_sch
Comment (alexandruradovici) [inline docs/project/2024/ionut.boghici/project.kicad_sch:1]: Please move the cicada files to the project's repository.

Comment (illusion2626) [reply docs/project/2024/ionut.boghici/project.kicad_sch:1]: I'm sorry, can you please clarify?

Comment (alexandruradovici) [reply docs/project/2024/ionut.boghici/project.kicad_sch:1]: Please move the kicad files to your project's repository. Fillow the discussion on Teans.

**Comment by alexandruradovici** [comment]
> > Please move the kicad files to your project's repository.
> 
> 
> 
> I moved them to my project's repository. They're in a zip file, let me know if I should add them separately.

Please do not upload zip files in repositories, make a folder with them.

File: docs/project/2024/ionut.boghici/Kicad Schematic.PNG
Comment (alexandruradovici) [inline docs/project/2024/ionut.boghici/Kicad Schematic.PNG:1]: Please do not use spaces in filenames.

File: docs/project/2024/ionut.boghici/project.kicad_sch
Comment (alexandruradovici) [reply docs/project/2024/ionut.boghici/project.kicad_sch:1]: This is a kicad file, please move it.

File: docs/project/2024/ionut.boghici/~project.kicad_sch.lck
Comment (alexandruradovici) [inline docs/project/2024/ionut.boghici/~project.kicad_sch.lck:1]: This is a kicad file, please move it.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/ionut.boghici/Kicad Schematic.PNG
Comment (illusion2626) [reply docs/project/2024/ionut.boghici/Kicad Schematic.PNG:1]: Changed it to have an underscore instead

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**